packages/vue/src/components/authenticator.vue

```diff
@@ -321,7 +307,7 @@ const emit = defineEmits([
 ]);
 
 const s = useInterpret(authMachine, {
-  devTools: process.env.NODE_ENV === 'development',
+  devTools: process.env.NODE_ENV !== 'development',
```

Comment:
This shouldn't be enabled for production should it? What did you experience?

```suggestion
  devTools: process.env.NODE_ENV === 'development',
```

Reply:
Good catch, I'll fix this. I changed this because when you build locally it always thinks it's in production for some reason.